### PR TITLE
Add support for datetime fields in user models

### PR DESCRIPTION
**Pull request by Andrew-Dickinson**
Looks like the Django custom `JSONEncoder` was only being used to compute signatures, but not to compute the actual payload sent via the webhook. 

This PR fixes that and adds a date field to the test model to expose the bug

### CODE REVIEW

**Review by danihodovic — APPROVED** [review]
Good stuff Andrew

**Comment by Andrew-Dickinson** [comment]
Formatter is passing on my machine, tests also. I think the CI failure came from missing migration files. Added them, waiting for your approval to re-run

**Pull request merged**
